examples/seismic/acoustic/operators.py

```diff
@@ -10,10 +10,19 @@ def laplacian(field, m, s, kernel):
     order in time formulation, the 4th order time derivative is replaced by a
     double laplacian:
     H = (laplacian + s**2/12 laplacian(1/m*laplacian))
-    :param field: Symbolic TimeFunction object, solution to be computed
-    :param m: square slowness
-    :param s: symbol for the time-step
-    :return: H
+
+    Parameters
+    ----------
+    field : TimeFunction
+        The computed solution.
+    m : float
+        Square slowness.
+    s :
```

Comment:
@mloubout is `s` a TimeDimension ? a Dimension? a SteppingDimension?

Reply:
`s` is the spacing of the TIme Dimension `grid.time_dim.spacing` or its value, `model.critical_dt`